include/oc_chef_wm.hrl

```diff
@@ -64,20 +64,25 @@
 
 -type permission() :: create | delete | read | update.
 
--type container_name() :: cookbook |
-                          data |
-                          environment |
-                          node |
-                          role |
-                          client |
-                          container |
-                          sandbox |
-                          policies |
-                          user |
-                          organization.
-
--type auth_tuple() :: {object, object_id(), permission()} |
-                      {container, container_name(), permission()}.
+-type container_name() :: cookbook
+                        | data
+                        | environment
+                        | node
+                        | role
+                        | client
+                        | container
+                        | sandbox
+                        | policies
+                        | user
+                        | organization
+                        | group
+                        | cookbook_artifact
+                        | policy_group.
+
+-type auth_tuple() ::
+        {create_in_container, container_name()}
```

Comment:
I'm wondering if we're trying to use this `auth_tuple` for multiple things where we shouldn't be. These don't seem like they go together.